lib/spack/spack/test/cmd/compiler.py

```diff
@@ -247,3 +248,86 @@ def test_compiler_list_empty(no_compilers_yaml, working_env, compilers_dir):
     out = compiler("list")
     assert not out
     assert compiler.returncode == 0
+
+
+def test_compilers_shows_packages_yaml(no_compilers_yaml, working_env, compilers_dir):
```

Comment:
The two tests being added in this file are almost equal. Can't we gather everything in one parametric test?

Reply:
We can -- it's not exactly pretty, but I've done it.